en/PlanView/PlanView.md

```diff
@@ -11,33 +11,39 @@ The image above shows a simple mission which starts with a takeoff, flies throug
 The steps to creating a missions are:
 
 1. Change to Plan View
-2. Add commands to the mission and edit as needed
-3. Send the mission to the vehicle
+2. Add waypoints to the mission and edit as needed
+3. Upload: Send the mission to the vehicle
```

Comment:
Either leave as it was or change to "Upload the mission to the vehicle" - there is no benefit to clarifying upload vs send as this does.

Reply:
got it! Thanks